libraries/ui-library-angular/src/lib/sidebar/active-sidebar-item-group.directive.ts

```diff
@@ -0,0 +1,26 @@
+import { ContentChildren, Directive, HostBinding, HostListener, inject, QueryList } from '@angular/core';
+import { RouterLinkActive } from '@angular/router';
+import { SixSidebarItem, SixSidebarItemGroup } from '../stencil-generated/components';
+
+/**
+ * This directive opens the six-sidbar-item-group bases on the active route
+ */
+@Directive({
+  selector: 'six-sidebar-item-group',
```

Comment:
Not sure about this, but I think we should make this directive optional. As is, the directive is applied all the time.